build/Program.fs

```diff
@@ -79,9 +98,12 @@ let listIntegrationTests() =
 let buildSdk() =
     cleanSdk()
     restoreSdk()
-    printfn "Building Pulumi SDK"
-    if Shell.Exec("dotnet", "build --configuration Release", sdk) <> 0
-    then failwith "build failed"
+    match findGoSDKVersion with
+    | None -> failwith "Could not find the Pulumi SDK version in go.mod"
+    | Some(version) ->
+        printfn "Building Pulumi SDK"
+        if Shell.Exec("dotnet", "build --configuration Release -p:PulumiSDKVersion=" + version, sdk) <> 0

```

Comment:
```suggestion
        if Shell.Exec("dotnet", "build --configuration Release -p:PulumiSdkVersion=" + version, sdk) <> 0

```

Dotnet style is _not_ to uppercase all letters of acronyms (e.g. the HTTP client class is `HttpClient`)